introduction.tex

```diff
@@ -7,6 +19,14 @@ \section{Introduction}
 to receive all the \gls{SNF} from EU nations to create \gls{MOX}.
 The \gls{MOX} created will fuel French transition to a \gls{SFR} fleet
 and may allow France to avoid building additional \glspl{LWR}.
+
+Past research focuses only on France, where additional \glspl{LWR},
```

Comment:
"Past research focuses only on France" . This statement is too broad to be correct. Perhaps what you're trying to say is "Past research which focuses solely on France has made the assumptions that additional LWRs .... '